### PR TITLE
Intermediate JavaScript Nanodegree Project 1 Dinosaurs #1

### PR DESCRIPTION
**Pull request by lwb2021**
# Project: Dinosaurs

## Video
https://user-images.githubusercontent.com/64982612/186610517-48dbefaa-603f-496a-9b0c-3623f99bd073.mov

### CODE REVIEW

**Review by Martian2Lee — CHANGES_REQUESTED** [review]
please update pr description following Leroy's example

File: index.html [hunk 1]
```diff
@@ -1,48 +1,94 @@
...
+        <label
+          ><input
+            id="weight"
+            class="form-field__full"
+            type="number"
+            name="weight"
+            min="0"
+            max="400"
+            required
+          />lbs</label
```
Comment (leemoispace) [inline index.html]: Maybe put weight unit in front of the input area to keep the same sequence logic like the height units.

File: app.js [hunk 2]
```diff
@@ -1,32 +1,218 @@
...
+}
+
+// Create weight compare method
+function compareWeight(dino) {
+  return dino.weight - humanInfo.weight;
+}
+
+// Create diet compare method
+function compareDiet(dino) {
+  return dino.diet;
```
Comment (leemoispace) [inline app.js:220]: very concise and clever way to make compare function.

File: app.js [hunk 3]
```diff
@@ -1,32 +1,218 @@
...
+  this.comparedDetails = humanInfo.comparedDetails;
+  this.feetHeight = humanInfo.feetHeight;
+  this.inchHeight = humanInfo.inchHeight;
+  this.weight = humanInfo.weight;
+  this.diet = humanInfo.diet;
+  this.url = humanInfo.url;
+}
+
+// Use IIFE to get human data from form
+(function () {
```
Comment (leemoispace) [inline app.js:144]: good, here i learnt how to use IIFE to get data.

File: app.js [hunk 4]
```diff
@@ -1,32 +1,218 @@
...
+  for (const dino of rawDinos) {
+    const heightDiff = compareHeight(dino);
+    const weightDiff = compareWeight(dino);
+    const dietDiff = compareDiet(dino);
+    dino.comparedDetails =
+      dino.species === "Pigeon"
+        ? `<h3>Species: ${dino.species}</h3> <br> 
+           All birds are Dinosaurs.`
+        : `
+      <h3>Species: ${dino.species}</h3> <br>
```
Comment (leemoispace) [inline app.js:102]: reduce some repeat title contents maybe better.

File: index.html (hunk 1 above)
Comment (lwb2021) [reply index.html]: fixed.

File: app.js (hunk 4 above)
Comment (lwb2021) [reply app.js:102]: can you tell me a bit more so I can follow your suggestion better since I am not sure which part is repetitive here?

**Pull request merged**